src/App/Fossa/VendoredDependency.hs

```diff
@@ -9,6 +9,10 @@ module App.Fossa.VendoredDependency (
   compressFile,
   hashFile,
   dedupVendoredDeps,
+  skippedDepsDebugLog,
```

Comment:
I moved `SkippableDeps` and `NeedScanningDeps` here as we're going to need them here when we implement skipping for archive uploads. It's also a good spot for them even before we do that